lib/Parser/parse.h

```diff
@@ -506,6 +512,26 @@ class Parser
     }
 
 public:
+    IdentPtrList* GetRequestedModulesList();
```

Comment:
const is against our coding conventions; I believe the argument is that we don't use const objects and so putting it on small accessors gains us nothing but ensure those accessors don't modify any fields, which is trivially obvious from the source of the accessor itself.